seastar.yaml

```diff
@@ -56,10 +56,8 @@ pipeline:
     with:
       opts: |
         -DSeastar_TESTING=OFF \
-        -DCMAKE_CXX_STANDARD=20 \
-        -DPROTOBUF_SOURCE=SYSTEM \
         -DBUILD_SHARED_LIBS=ON \
-        -DWITH_SSL=system
+        -DBUILD_STATIC_LIBS=ON
```

Comment:
Why the switch to static build? The other options didn't do anything, but this one ensures using system libraries where available.

Reply:
I tried that and the static package was empty when using `split/static` so removed.